config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+type Config struct {
+	Name		string			`mapstructure:"name"`
+	CliRoot	string			`mapstructure:"homeclient"`
+	Init		*InitConfig		`mapstructure:"init"`
+	GenTx		*GenTxConfig	`mapstructure:"gentx"`
+}
+
+func DefaultConfig() *Config {
+	return &Config{
+		Name: "",
+		CliRoot:	"",
+		Init: DefaultInitConfig(),
+		GenTx: DefaultGenTxConfig(),
+	}
+}
+
+type InitConfig struct {
+	ChainID		string			`mapstructure:"chainid"`
+	GenTxs		bool			`mapstructure:"gentxs"`
+	GenTxsDir	string			`mapstructure:"gentxsdir"`
+	Overwrite	bool			`mapstructure:"overwrite"`
```

Comment:
I'm confused as to how `config` is uses - maybe add a comment at the top of this file... also what is `mapstucture` used for?

Reply:
Completely thrown out as discussed.